packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -165,14 +163,24 @@ const _generateStyles = (
         },
         restingInline: {
             color: defaultTextColor,
+            // TODO(WB-1521): Update text decoration to the dashed underline
+            // after the Link audit.
+            // textDecoration: "underline currentcolor solid 1px",
```

Comment:
**question:** should this be `2px`?

Reply:
Hmm... I actually think it should be `auto`.